deploy/bin/import-docker-worker-secrets

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# This script reads docker-worker store in passwordstore and writes
+# up deploy.json.
+#
+# Use: ./import-docker-worker-secrets --passphrase=<your-passphrase>
+#
+# Notice you must have taskcluster passwordstore cloned and configured.
+# See ssh://gitolite3@git-internal.mozilla.org/taskcluster/secrets.git
+# for details.
+
+export PASSWORD_STORE_GPG_OPTS="$*"
+
+base_dir=/tmp
+
+pass show aws/workers-key > $base_dir/docker-worker.key
```

Comment:
Is `set -e` enabled here?  If this (or the later pass invocations) fails, the script should fail..

Reply:
Fixed.